webapp/app/queries/routes.py

```diff
@@ -36,6 +36,10 @@
     jsonify_query_result,
     parse_date_range_argument,
 )
+from cropcore.generate_synthetic_data import (
+    generate_batches_in_farm,
```

Comment:
Does this get used somewhere?

Reply:
Eeek well spotted!! - that function+endpoint is the entire point of the PR (to get stuff into the unity demo), but somehow the endpoint function must have got lost when I was moving things around...   Restoring now.